clearml_agent/backend_api/config/default/agent.conf

```diff
@@ -325,4 +325,9 @@
     # into the file specified in CLEARML_CUSTOM_BUILD_OUTPUT, the agent will emit a warning and continue with the
     # standard flow.
     custom_build_script: ""
+
+    # when set to false if the agent encounters an exception, 
+    # it will catch and log the exception and continue running.
+    # When set to true the exception will be propogated.
+    crash_on_exception: false
```

Comment:
One last comment, I promise! 🙂 
Please change this to:
```
    # Crash on exception: by default when encountering an exception while running a task,
    # the agent will catch the exception, log it and continue running.
    # Set this to `true` to propagate the exception and crash the agent.
    # crash_on_exception: true
```
So that by default it will not be defined, and the user has an example he can simply un-comment in order to change behavior.

Reply:
No worries, thanks for the comments! 😄 

I adjusted the config file 👍